styleguide/derek/global-components/filter-bar/_filter-bar.scss

```diff
@@ -138,3 +138,163 @@
     margin: .5rem auto 0;
   }
 }
+
+// Start of accordion style side bar
+.rsFilter-sideBar {
+  @include make-sm-column(2);
```

Comment:
Is 2 columns too narrow here? We're all but guaranteed to get wrapping text like this: 

![image](https://user-images.githubusercontent.com/5263371/31191207-0dc2a090-a903-11e7-8a22-8e7816be445e.png)

I realize that may require a bit of coordination with @amberRrucker on the main resource content area as well, but I just feel like we're trying to cram too much text in too small of a region.

Reply:
This is the current behavior of the side bar. It's a col (2)

Reply:
If this task is to cleanup the sidebar though, I don't necessarily see solving this as outside of the scope.